src/openrct2-ui/windows/Guest.cpp

```diff
@@ -2067,6 +2067,11 @@ void window_guest_debug_paint(rct_window* w, rct_drawpixelinfo* dpi)
     auto peep = GET_PEEP(w->number);
     auto x = w->x + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].left + 4;
     auto y = w->y + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].top + 4;
+    {
+        set_format_arg(0, uint32_t, peep->sprite_index);
+        gfx_draw_string_left(dpi, STR_PEEP_DEBUG_SPRITE_INDEX, gCommonFromatArgs, 0 x, y);
```

Comment:
Typo!
```suggestion
        gfx_draw_string_left(dpi, STR_PEEP_DEBUG_SPRITE_INDEX, gCommonFormatArgs, 0 x, y);
```

Reply:
wow it took me an embarrassingly long amount of time to spot the difference. ill get it fixed. thanks for pointing that out